glide.lock

```diff
@@ -1,22 +1,28 @@
-hash: 5fc263bfacc703e0a599166ccae968891dd53d7b4e42a3d8356b3194e45833bb
-updated: 2017-12-18T16:46:06.7851579+05:30
+hash: c48583fd7daa415c986e22638c3036b02f22698a0f5d5f583ce1b6c06833a3b3
+updated: 2018-02-28T14:36:47.873451024Z
```

Comment:
This is a good point and something I sort of overlooked. We should only see changes to packages that are brought in by the new dependency(s). Node that glide itself probably made these changes to glide.lock when glide.yaml was updated. But still it would be good to minimize any changes to the dependencies.

(I'm not going to bring up the fact that glide is no longer the favored approach to dependencies in golang here though. :-))

Reply:
Not sure how to address this one. Will accept any guidance here.

Reply:
Will try to explain in more detailed comments.

Reply:
(and yeah, we should probably move to "dep" at some point...)